.github/workflows/publishing.yml

```
@@ -0,0 +1,130 @@
+
+      - name: Test PyPI Connection
+        run: |
+          STATUS=$(curl -s -o /dev/null -w "%{http_code}" https://pypi.org/)
+          if [ "$STATUS" != "200" ]; then
+            echo "PyPI connection failed with status $STATUS"
+            exit 1
+          fi
+
+  check-version:
+    needs: test-pypl-connection
+    if: ${{ needs.test-pypl-connection.outputs.status == '200' }}
```

actionlint (1.7.12)

[error] property "status" is not defined in object type {}

(expression)